CHANGELOG.md

```diff
@@ -5,6 +5,43 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [v3.110.0](https://github.com/buildkite/agent/tree/v3.110.0) (2025-10-22)
+[Full Changelog](https://github.com/buildkite/agent/compare/v3.109.1...v3.110.0)
+
+### Added
+- Configurable chunks interval [#3521](https://github.com/buildkite/agent/pull/3521) (@catkins)
+- Inject OpenTelemetry context to all child processes [#3548](https://github.com/buildkite/agent/pull/3548) (@zhming0)
```

Comment:
Do we want to call out the potential breaking change aspect of this change? Since we are adding a number of common Otel environment variable to child processes, there is a chance that customer has already set those variables (somehow)

Reply:
Done